tables/nl_comp8.tbl

```diff
@@ -0,0 +1,11 @@
+#-index-name: Dutch, computer, 8-dot
+#-display-name: Dutch 8-dot computer braille draft
+
+#+locale:nl
+#+type:computer
+#+dots:8
+#+direction:both
+#+version:2019.11.15
+
+include nl-comp8.utb
```

Comment:
Can you please combine nl-comp8.utb and nl_comp8.tbl into a single file?

Reply:
I assume you want the tbl file to remain?

Reply:
Actually, maybe it's better to keep the utb file and drop the tbl file because as far as the NEWS file is concerned none of the current tbl files even exist. This way we keep some consistency.